presto-common/src/main/java/com/facebook/presto/common/RuntimeUnit.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.common;
+
+import com.facebook.drift.annotations.ThriftEnum;
+import com.facebook.drift.annotations.ThriftEnumValue;
+
+@ThriftEnum
+public enum RuntimeUnit
+{
+    NONE(0), NANO(1), BYTE(2);
+
+    private final int value;
+
+    RuntimeUnit(int value)
+    {
+        this.value = value;
```

Comment:
`requireNonNull`

Reply:
This parameter is `int` and cannot be `null` right?

Reply:
missed that .. you are right !